### ISSUE TITLE
Source 'https://rails-assets.org' results in Could not find gem

### ISSUE THREAD

**Issue by stiff** [MaximeD/gem_updater#13]
In gemfile I have this block:

```ruby
...
source 'https://rails-assets.org' do
  gem 'rails-assets-bacon'
end
```

`bundle update` works just fine, but `gem_update` gives the error:
```
WARN: Unresolved specs during Gem::Specification.reset:
      mini_portile (~> 0.6.0)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
~/.rvm/gems/ruby-2.2.0/gems/bundler-1.8.3/lib/bundler/resolver.rb:369:in `resolve': Could not find gem 'rails-assets-bacon (>= 0) ruby' in rubygems repository https://rails-assets.org/. (Bundler::GemNotFound)
Source does not contain any versions of 'rails-assets-bacon (>= 0) ruby'
  from ~/.rvm/gems/ruby-2.2.0/gems/bundler-1.8.3/lib/bundler/resolver.rb:167:in `start'
  from ~/.rvm/gems/ruby-2.2.0/gems/bundler-1.8.3/lib/bundler/resolver.rb:129:in `resolve'
  from ~/.rvm/gems/ruby-2.2.0/gems/bundler-1.8.3/lib/bundler/definition.rb:193:in `resolve'
  from ~/.rvm/gems/ruby-2.2.0/gems/bundler-1.8.3/lib/bundler/definition.rb:132:in `specs'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.1.0/lib/gem_updater/gem_file.rb:10:in `initialize'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.1.0/lib/gem_updater.rb:9:in `new'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.1.0/lib/gem_updater.rb:9:in `initialize'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.1.0/bin/gem_update:8:in `new'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.1.0/bin/gem_update:8:in `<top (required)>'
  from ~/.rvm/gems/ruby-2.2.0/bin/gem_update:23:in `load'
  from ~/.rvm/gems/ruby-2.2.0/bin/gem_update:23:in `<main>'
```

**Comment by MaximeD** [comment]
Hello, thanks for reporting it, there is indeed a bug when using other sources. I'll have a look.

**Comment by stiff** [comment]
Thanks!

**Comment by MaximeD** [comment]
Hi, support for rails assets has been integrated in version 0.2.0

**Comment by stiff** [comment]
Great, but now I still have another exception :)
```
Looking for a changelog in http://wiki.github.com/wvanbergen/chunky_png
~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/open-uri.rb:224:in `open_loop': redirection forbidden: http://wiki.github.com/wvanbergen/chunky_png -> https://wiki.github.com/wvanbergen/chunky_png (RuntimeError)
  from ~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/open-uri.rb:150:in `open_uri'
  from ~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/open-uri.rb:716:in `open'
  from ~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/open-uri.rb:30:in `open'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.2.0/lib/gem_updater/source_page_parser.rb:23:in `changelog'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.2.0/lib/gem_updater.rb:29:in `block in update!'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.2.0/lib/gem_updater.rb:25:in `each'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.2.0/lib/gem_updater.rb:25:in `update!'
  from ~/.rvm/gems/ruby-2.2.0/gems/gem_updater-0.2.0/bin/gem_update:9:in `<top (required)>'
  from ~/.rvm/gems/ruby-2.2.0/bin/gem_update:23:in `load'
  from ~/.rvm/gems/ruby-2.2.0/bin/gem_update:23:in `<main>'
```

**Comment by MaximeD** [comment]
Oh my :) Sorry about that I'll make another issue for it.

**Comment by MaximeD** [comment]
Hello, this was fix by #18 and latest release (version 0.3.0) will include the fix, I hope everything will run smoothly for you now ;)

**Comment by stiff** [comment]
Yes, thank you very much.
